android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -14,263 +14,256 @@ import java.util.UUID
 @ReactModule(name = PlayerModule.name)
 class PlayerModule(private val context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
     /**
-     * In-memory mapping of `id` <-> `Player`.
+     * In-memory mapping of `nativeId` strings and `Player` instances.
      */
     private var registry: MutableMap<String, Player> = mutableMapOf()
 
     /**
-     * Exported module name to JS.
+     * JS exported module name.
      */
     companion object {
         const val name = "PlayerModule"
     }
     override fun getName() = PlayerModule.name
 
     /**
-     * Fetch the `Player` instance with id equal to `playerId` inside this module's `registry`.
-     * @param playerId Target player to look inside registry.
+     * Fetches the `Player` instance associated with `nativeId` from the internal registry.
+     * @param nativeId `Player` instance ID.
+     * @return The associated `Player` instance or `null`.
      */
-    fun getPlayer(playerId: String?): Player? {
-        if (playerId == null) {
+    fun getPlayer(nativeId: String?): Player? {
```

Comment:
These changes are related to the way I'm architecting the relationship between the SDKs native objects (`Player`, `Source`, and `DrmConfig`) in the JS side.

The idea is that each instance of any of these types in JS will have a [`nativeId`](https://github.com/bitmovin/bitmovin-player-react-native/blob/feature/drm-playback/src/nativeInstance.ts#L28) property to access its native counterpart that will be evaluated during initialization. And then we're able to start relating them on the API just using this property, for example:

```typescript
// Generates a unique `nativeId` for `Source` during runtime.
const source = new Source({
  /** config **/
});

// Uses the underlying `nativeId` to access both the `Player` and `Source` native objects from Kotlin/Swift.
player.load(source);
```

And until the DRM feature, `Player` was the only native instance with this capability, and its `nativeId` property was referenced as `playerId` on the native side. But once `Source` and `DrmConfig` objects were added to the JS API in this PR, I updated the references to `nativeId` just to keep them standardized.

Reply:
Thanks for the explanation!
I see, the changes make sense, but unfortunate that this ended up in the DRM feature PR. It could have been pre-work for this, to make reviews a bit easier and faster.
No need to change anything but please keep in mind for the future changes.

Reply:
Ok, I will. Thanks for the feedback!

I'll start submitting changes in smaller PRs from now on. I'll just need to do some work breaking up the existing issues into smaller tasks, but it should be fine.